src/main/java/network/brightspots/rcv/Utils.java

```diff
@@ -83,4 +83,8 @@ static String getUserName() {
     }
     return user;
   }
+
+  static String[] splitByNewline(String s) {
+    return s.split("\\s*\\r?\\n\\s*");
```

Comment:
One last quirk with this: if you make the first line or first few lines blank, it'll count it as a null alias (happens in both the original alias box and in the edit pop-up).

Reply:
Fixed!